### ISSUE TITLE
Sir,new problem

### ISSUE THREAD

**Issue by z646587572** [jamesmontemagno/Xamarin.Forms-Monkeys#10]
in your Monkeys-CarouselView project，
i must delete "[assembly: XamlCompilation(XamlCompilationOptions.Compile)]" in AssemblyInfo.cs.
 this can work.

if i didn't delete that row, the running will be wrong.

is it new version xamarin problem?

**Comment by shah-smit** [comment]
@z646587572  Yes, I did experience the same. I think thats the issue. And Thanks I was trying to resolve the issue of the build and this soln helped me.
